Code/Engine/RendererCore/Lights/Implementation/DirectionalLightComponent.cpp

```diff
@@ -14,8 +14,8 @@ EZ_BEGIN_COMPONENT_TYPE(ezDirectionalLightComponent, 3, ezComponentMode::Static)
 {
   EZ_BEGIN_PROPERTIES
   {
-    EZ_ACCESSOR_PROPERTY("NumCascades", GetNumCascades, SetNumCascades)->AddAttributes(new ezClampValueAttribute(1, 4), new ezDefaultValueAttribute(3)),
-    EZ_ACCESSOR_PROPERTY("MinShadowRange", GetMinShadowRange, SetMinShadowRange)->AddAttributes(new ezClampValueAttribute(0.0f, ezVariant()), new ezDefaultValueAttribute(50.0f), new ezSuffixAttribute(" m")),
+    EZ_ACCESSOR_PROPERTY("NumCascades", GetNumCascades, SetNumCascades)->AddAttributes(new ezClampValueAttribute(1, 4), new ezDefaultValueAttribute(2)),
```

Comment:
Why do we change those default values?

Reply:
The default costs A LOT of performance and in most games these would be "high end" settings. In my tests I could also not see a difference between 2 and 3 layers. This way the user doesn't accidentally waste performance when it is not needed for their use case.